formbuilder/config.js

```diff
@@ -99,5 +99,55 @@ export default {
       },
       formData: {}
     },
+    {
+      id: "select",
```

Comment:
Hi, the select has been removed a few months ago since it has the same use as the choice list. I'm not sure what we should do with this idea, since I personally like better this one, so maybe we should replace the former with the latter.